File: RootTools2/src/main/java/com/rarnu/tools/neo/data/Onekey.java
package com.rarnu.tools.neo.data;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by rarnu on 9/7/16.
 */
public class Onekey {

    public String pkgName = "";
    public String[] disabledComponents = null;

    public Onekey(String pkg, String str) {
        this.pkgName = pkg;
        try {
            String[] strs = str.split("\n");
            disabledComponents = strs;
        } catch (Exception e) {

        }
    }

}

File: RootTools2/src/main/kotlin/com/rarnu/tools/neo/data/Onekey.kt
package com.rarnu.tools.neo.data

/**
 * Created by rarnu on 9/7/16.
 */
class Onekey(str: String) {
    var disabledComponents: Array<String>? = null
    init {
        try {
            val strs = str.split("\n").dropLastWhile(String::isEmpty).toTypedArray()
            disabledComponents = strs
        } catch (e: Exception) {

        }
    }

}
